packages/site/src/data/graphql/libraries.ts

```diff
@@ -1,26 +1,490 @@
+		image: "https://github.com/graphql-dotnet.png",
+		tags: ["Servers"],
+	},
+	{
+		name: "GraphQL Hooks",
+		author: "NearForm",
+		gitHubRepo: "https://github.com/nearform/graphql-hooks",
+		npmPackage: "",
+		href: "https://github.com/nearform/graphql-hooks",
+		description: "Minimal hooks-first GraphQL client.",
+		image: "https://github.com/nearform.png",
+		tags: ["Clients"],
+	},
+	{
+		name: "GraphQL-request",
+		author: "Prisma Labs",
+		gitHubRepo: "https://github.com/prisma-labs/graphql-request",
+		npmPackage: "graphql-request",
+		href: "https://github.com/prisma-labs/graphql-request",
+		description:
+			"Minimal GraphQL client supporting Node and browsers for scripts or simple apps",
+		image: "https://github.com/prisma-labs.png",
+		tags: ["Clients"],
+	},
+	{
+		name: "React Query",
+		author: "TanStack",
+		gitHubRepo: "https://github.com/tanstack/query",
+		npmPackage: "@tanstack/react-query",
+		href: "https://tanstack.com/query/v4/",
+		description:
+			"Powerful asynchronous state management for TS/JS, React, Solid, Vue and Svelte",
+		image: "https://github.com/TanStack.png",
+		tags: ["Clients"],
+	},
+	{
+		name: "URQL",
+		author: "Formidable",
+		gitHubRepo: "https://github.com/FormidableLabs/urql",
+		npmPackage: "urql",
+		href: "https://formidable.com/open-source/urql/",
+		description: "A highly customisable and versatile GraphQL client",
+		image:
+			"https://raw.githubusercontent.com/FormidableLabs/urql/main/packages/site/src/assets/sidebar-badge.svg",
+		tags: ["Clients"],
+	},
+	{
+		name: "Apollo Kotlin",
+		author: "Apollo GraphQL",
+		gitHubRepo: "https://github.com/apollographql/apollo-kotlin",
+		npmPackage: "",
+		href: "https://www.apollographql.com/docs/kotlin/",
+		description:
+			"A strongly-typed, caching GraphQL client for Java and Kotlin multiplatform",
+		image: "https://github.com/apollographql.png",
+		tags: ["Clients"],
+	},
+	{
+		name: "Apollo Client",
+		author: "Apollo GraphQL",
+		gitHubRepo: "https://github.com/apollographql/apollo-client",
+		npmPackage: "@apollo/client",
+		href: "https://www.apollographql.com/docs/react/",
+		description:
+			"Apollo Client is a comprehensive state management library for JavaScript that enables you to manage both local and remote data with GraphQL. Use it to fetch, cache, and modify application data, all while automatically updating your UI.",
+		image: "https://github.com/apollographql.png",
+		tags: ["Clients"],
+	},
+	{
+		name: "Apollo iOS",
+		author: "Apollo GraphQL",
+		gitHubRepo: "https://github.com/apollographql/apollo-ios",
+		npmPackage: "",
+		href: "https://www.apollographql.com/docs/ios/",
+		description:
+			"Apollo iOS is a strongly-typed, caching GraphQL client, written in Swift.",
+		image: "https://github.com/apollographql.png",
+		tags: ["Clients"],
+	},
+	{
+		name: "Swift GraphQL",
+		author: "Matic Zavadlal",
+		gitHubRepo: "https://github.com/maticzav/swift-graphql",
+		npmPackage: "",
+		href: "swift-graphql.com",
```

Comment:
Noticed this goes to `http://localhost:3000/categories/swift-graphql.com `when clicked. 

Looks like it needs to be set as  `https://swift-graphql.com`